changelog/fragments/ansible-helm-ssc.yaml

```
@@ -0,0 +1,44 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (ansible/v1)(helm/v1) Add SecurityContext to now allow root privileges.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "addition"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (ansible/v1)(helm/v1) **(Optional)** Add SecurityContext to now allow root privileges.
```

All `v1.y` migrations are optional.
```suggestion
      header: (ansible/v1, helm/v1) Add `securityContext`'s to your manager's Deployment.
```